charts/csi-vxflexos/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
-appVersion: "2.10.0"
+appVersion: "2.10.2"
```

Comment:
`appVersion` should be 2.10.1 since this is in sync with the driver version not the helm chart version.

Reply:
fixed.